<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentP</class>
 <widget class="QWidget" name="studentP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="studentStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>850</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="studentMain">
    <widget class="QPushButton" name="moveAtten">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>660</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>출결</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveMessage">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메시지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveCalendar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>490</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>일정</string>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>학생 이름</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="moveChat">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>채팅</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>님</string>
     </property>
    </widget>
    <widget class="QLabel" name="chatA">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="studentMessage">
    <widget class="QPushButton" name="sendMessage">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>590</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메시지 보내기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainHome_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>700</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
    <widget class="QComboBox" name="teacherSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>이상복</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>류홍걸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>조동현</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="studentName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>학생 이름</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>보내는 이 :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="message">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>390</width>
       <height>260</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>교수님에게 메시지 보내기</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>교수님,</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="studentChat">
    <widget class="QTextBrowser" name="chatting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>431</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mainHome_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>760</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="chatText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>입력할 채팅</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chat">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>700</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>채팅 보내기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>교수님</string>
     </property>
    </widget>
    <widget class="QComboBox" name="teacherSet_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>이상복</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>류홍걸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>조동현</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="chatLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>700</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>채팅 로그 보기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="studentCalendar">
    <widget class="QCalendarWidget" name="sCalendar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>350</width>
       <height>400</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="studentAddTask">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>770</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>일정 추가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>710</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>이름</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainHome">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>770</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="sTasks">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>470</y>
       <width>351</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="task">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>710</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>일정 내용</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>670</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>일정 추가하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="studentAtten">
    <widget class="QPushButton" name="attenButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>입실</string>
     </property>
    </widget>
    <widget class="QPushButton" name="outingButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>530</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>외출</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>입실 시간</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>퇴실 시간</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userName_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>학생 이름</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="todayDate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>오늘 날짜</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="attenTime">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>__ : __</string>
     </property>
    </widget>
    <widget class="QLabel" name="backTime">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>__ : __</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainHome_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>620</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>메인 홈</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
